JavaExample/app/build.gradle

```diff
@@ -44,10 +44,10 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'com.android.support:appcompat-v7:28.0.0'
```

Comment:
Not a huge thing, but it's worth considering to migrate all support libraries to androidx sometime in the future. (https://developer.android.com/jetpack/androidx/migrate).